roles/atomic_upgrade/tasks/main.yml

```diff
@@ -34,10 +34,9 @@
 
 - block:
 
-    - name: Workaround BZ1290281
-      selinux:
-        policy: targeted
-        state: permissive
+    - name: Workaround old atomic command bug: failure b/c /tmp is a symlink
```

Comment:
No worries, there probably isn't a good way to word it when neither of us really understands the underlying problem 😁